docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -82,7 +82,25 @@ This configures the underlying OTel Collector with a pipeline for traces. The re
     http-backend      Ready     44s
     ```
 
-### 2. Switch the protocol to HTTP
+### Step 2. Enable Istio Tracing
```

Comment:
If you want to use the word `Step`, please add it to point **1. Create a TracePipeline with an output.** Otherwise, please remove the word from the remaining points. I recommend the second option :)

Reply:
Nina introduced it.. want to keep ot consistent with logging guide